src/sales/checkout-options.md

```diff
@@ -11,17 +11,45 @@ _[Checkout Options]({% link configuration/sales/checkout.md %})_{:.ee-only}
 
 1. On the _Admin_ sidebar, go to **Stores** > _Settings_ > **Configuration**.
 
-1. On the left panel, expand **Sales** and choose **Checkout**.
+1. On the left panel, expand **Sales**.
 
-1. Expand ![]({% link images/images/btn-expand.png %}) the **Checkout Options** section.
+1. Set any of the following options you require.
 
-1. Set the checkout options:
+### Checkout Options
```

Comment:
These aren't nested and will break your numbering. Also, you can't use subheads in a numbered list.

My recommendation would be to move these sections to the bottom of the page and revise the step to list the option "sections" with links to the list of items. I would also include a link to each section in the configuration reference page: https://docs.magento.com/user-guide/configuration/sales/checkout.html